aws/istio-ingress/overlays/cognito/ingress.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: extensions/v1beta1
+apiVersion: extensions/v1beta1 # networking.k8s.io/v1beta1
```

Comment:
Do you want to add a TODO(<link to github issue>) about updating this?

Reply:
@jlewi I will add an issue so that we remember to come back and fix them once with switch to Kubernete 1.14+. I'll add additional instruction on what to pay attention to in Role and ClusterRole definition.